nni/retiarii/converter/graph_gen.py

```diff
@@ -660,8 +660,9 @@ def _convert_module(self, script_module, module, module_name, module_python_name
             attrs = {
                 'mutation': 'repeat',
                 'label': module.label,
+                'depth': module.depth_choice,
+                'max_depth': module.max_depth,
```

Comment:
what is the difference between depth and max_depth?

Reply:
`depth` is the chosen depth, `max_depth` and `min_depth` is a depth range.